src/test/kotlin/ratpack/example/kotlin/SiteSpec.kt

```diff
@@ -0,0 +1,72 @@
+package ratpack.example.kotlin
+
+import com.google.common.io.CharStreams
+import org.jetbrains.spek.api.Spek
+import ratpack.test.MainClassApplicationUnderTest
+import java.io.InputStreamReader
+import kotlin.test.assertEquals
+
+class SiteSpec : Spek({
+
+    val aut = MainClassApplicationUnderTest(Main::class.java)
```

Comment:
I'm looking for something like Spock's `@AutoCleanup` `spock.lang.AutoCleanup` since a lot of Ratpack's test fixtures are `AutoCloseable`